pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -139,4 +141,22 @@ protected void cleanup() throws Exception {
                 .enableBatchIndexAcknowledgment(true)
                 .subscribe();
     }
+
+    @Test
+    public void testGetTxnID() throws Exception {
+        Awaitility.await().atMost(4, TimeUnit.SECONDS).until(()->{
+            try {
+                Transaction transaction = pulsarClient.newTransaction()
+                        .withTransactionTimeout(0, TimeUnit.SECONDS).build().get();
+            } catch (Exception e){
+                return false;
+            }
+            return true;
+        });
+        Transaction transaction = pulsarClient.newTransaction()
+                .withTransactionTimeout(0, TimeUnit.SECONDS).build().get();
+        TxnID txnID = transaction.getTxnID();
+        Assert.assertEquals(txnID.getLeastSigBits(), 1);
+        Assert.assertEquals(txnID.getMostSigBits(), 0);
```

Comment:
We will always get transaction ID (1,0) here? I mean we can just check the txnID is not null.

Reply:
In fact it should always be this